tests/check/check_test.go

```diff
@@ -43,7 +43,7 @@ func TestCheckMySQL(t *testing.T) {
 	testRunAll(t, "mysql")
 }
 
-func TestCheckLogs(t *testing.T) {
+func TestCheckLogsAndHeaders(t *testing.T) {
```

Comment:
Since headers are a second-class citizen of the public API contract, it feels more appropriate that these tests for headers should be part of a functional API test instead of here in the test that asserts certain logs are being emitted from the server.

Can we relocate these headers to a more appropriate location that coincides with the HTTP grpc-gateway suite of functional tests?

Reply:
Moved